apis/infix-en_GB/atrium-api-infix-en_GB-common/src/main/kotlin/ch/tutteli/atrium/api/infix/en_GB/listAssertions.kt

```diff
@@ -0,0 +1,39 @@
+package ch.tutteli.atrium.api.infix.en_GB
+
+import ch.tutteli.atrium.api.infix.en_GB.creating.list.get.builders.ListGetStep
+import ch.tutteli.atrium.creating.Expect
+import ch.tutteli.atrium.domain.builders.ExpectImpl
+
+/**
+ * Expects that the given [index] is within the bounds of the subject of the assertion (a [List]) and
+ * returns an [Expect] for the element at that position.
+ *
+ * @return The newly created [Expect] for the element at position [index].
+ * @throws AssertionError if the given [index] is out of bound.
+ *
+ * @since 0.9.0
```

Comment:
```suggestion

```
This functionality kind of existed in 0.8.0 as well, just in another API module. Therefore we don't add it.